modules/ROOT/pages/to-download-and-install-studio-lx.adoc

```diff
@@ -3,27 +3,39 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-To support some of Studio Themes in Linux, it is recommended to have GTK version 2 installed.
+== Prerequisites
 
-* Before downloading and launching Anypoint Studio, make sure you have the Java SE JDK 8 installed.
+* Ensure that you have the Java SE JDK 8 installed and configured.
+* To support some of Studio Themes in Linux, it is recommended to have GTK version 2 installed.
+* Download the Anypoint Studio installation file from the https://www.mulesoft.com/lp/dl/studio[Anypoint Studio Download Site].
+
+[NOTE]
+Anypoint Studio 7.x only supports Mule 4.x projects since the structure of the project, export format, XML and scripting language are different.
 
-To install Anypoint Studio:
 
-. Download Anypoint Studio from the https://www.mulesoft.com/lp/dl/studio[Anypoint Studio Download Site] and extract it.
+== Installation
+
 . Open the extracted Anypoint Studio.
 . Click OK to accept the default workspace. +
-When setting a custom path to your workspace, keep in mind that Anypoint Studio does not expand the `~` tilde used in Unix/Linux systems. It is recommended to use the absolute path when defining your workspace.
-+
-[NOTE]
---
-You cannot reuse a workspace used by Anypoint Studio 6.x or below. +
-If you choose to use an existing workspace, this must have been created using Anypoint Studio 7.0 or above.
---
+Use an absolute path when defining your workspace because Anypoint Studio does not expand the `~` tilde used in Unix/Linux systems. You cannot reuse a workspace used by Anypoint Studio 6.x or below. If you choose to use an existing workspace, this must have been created using Anypoint Studio 7.0 or above.
```

Comment:
You must use

Reply:
UNIX and Linux systems

Reply:
6.x or earlier.

However, you can omit this sentence in favor of the next, which basically reiterates it.

Reply:
If you choose to use an existing workspace, it must have been created using Anypoint Studio 7.0 or later.

Reply:
Question: Is this final sentence related to the Note above?
If they are the same thing, need we repeat this?

Reply:
They are not exactly the same, but one is a consequence of the other.
Studio 7.x only supports Mule 4.x projects, and at the same time, users cannot have Mule 3.x and Mule 4.x projects in the same workspace.  I think some users assumed that they could mix the projects and Studio 7.x would only pick up Mule 4.x projects, but that's not the case.

Reply:
I like making this shorter and go with:
```
You must Use an absolute path when defining your workspace because Anypoint Studio does not expand the `~` tilde used in Unix and Linux systems. If you choose to use an existing workspace, this must have been created using Anypoint Studio 7.0 or above.
```

Would it make sense to make them bullet points?
```
. Click *OK* to accept the default workspace. 
** You must Use an absolute path when defining your workspace because Anypoint Studio does not expand the `~` tilde used in Unix and Linux systems. 
** If you choose to use an existing workspace, this must have been created using Anypoint Studio 7.0 or above.
```